config/shared/activities.js

```diff
@@ -127,10 +128,16 @@ function getElementConfig(schemaId, type) {
 function getSiblingLevels(type) {
   const schemaId = getSchemaId(type);
   if (!schemaId) return [{ type }];
-  const levels = getOutlineLevels(schemaId);
-  const { level } = find(levels, { type }) || {};
-  if (!level) return [{ type }];
-  return filter(levels, { level });
+  const outline = getOutlineLevels(schemaId);
+  const activityConfig = find(outline, { type });
+  if (!activityConfig) return [{ type }];
```

Comment:
I carried this logic over from the previous implementation, but this case could only happen if the activity is either not in schema config (which I don't know should we even handle) or more likely if it's not an outline activity.
The latter case assumes only reorder between the same types of activities which is currently correct since containers are grouped per type but might be limiting in the future and also off if this would be used for non-type-grouped sub-containers in custom implementations...
Thoughts @ExtensionEngine/tailor ?